docs/ref/contrib/auth.txt

```diff
@@ -529,3 +534,14 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
    new user is created, and can be used to perform custom setup actions, such
    as setting the user's groups based on attributes in an LDAP directory.
    Returns the user object.
+
+
+.. class:: AllowInactiveUsersModelBackend
+
+   Same as :class:`~ModelBackend`, but does not reject inactive users by
+   default.
+
+   .. attribute:: AllowInactiveUsersModelBackend.allow_inactive_users
```

Comment:
I wouldn't list this separately. Above I would say, "doesn't reject inactive users because ~ModelBackend.allow_inactive_users`=True."